app/views/root/designprinciples.html.erb

```diff
@@ -180,9 +180,10 @@
             think about those people from the start.
            </p>
            <ul>
+             <li><a href="https://accessibility.blog.gov.uk/2016/05/16/what-we-mean-when-we-talk-about-accessibility-2/"></a>What we mean when we talk about accessibility, by&nbsp;Alistair&nbsp;Duggin</li>
```

Comment:
More nits, sorry! Looks like you'll need the closing </a> after the name of the article, but before the name of the author - to keep with the convention for the others.